### ISSUE TITLE
The default language should be selected in dropdown

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4227]
**Describe the bug**
In categories, the "Default language" dropdown, the deployment-language should be selected when creating a new category. It is now blank, although the deployment-language is actually set.

<img width="969" alt="Screenshot 2021-03-25 at 08 58 55" src="https://user-images.githubusercontent.com/8624777/112438563-577e7880-8d48-11eb-96e5-f56f186da73a.png">

**To Reproduce**
As logged in admin:
1. Go to /settings/categories/create
2. Look at the dropdown "Select default language for this category"
3. It is blank

**Expected behavior**
The default language of the deployment should be selected.

**Comment by helen-ndip** [comment]
Hi, @Angamanga as I am not a first-timer, I would like to work on this issue.  Can I this be assigned to me?

**Comment by Angamanga** [comment]
@helen-ndip Absolutely! Please go ahead! 🎉

**Comment by helen-ndip** [comment]
Hi, @Angamanga  hope you enjoyed your Easter holiday. I have been trying to replicate this issue over the weekend but noticed it has been fixed somehow already because I do get the default language on my instance. I went ahead and did a fix for the default language not showing on the sidebar instead. 

![image](https://user-images.githubusercontent.com/72542706/113701681-5cd8ad00-96d0-11eb-95bd-531fa9289bea.png)
Already fixed

![image](https://user-images.githubusercontent.com/72542706/113701863-9c06fe00-96d0-11eb-83cd-40b069925b0f.png)
I did  a fix for this instead

If this is ok, I would like to send a pull request.

**Comment by Angamanga** [comment]
@helen-ndip Hmm, interesting! Did you try creating a new Category? I see the default language when editing categories, but not when creating a new one: 
<img width="887" alt="Screenshot 2021-04-06 at 13 21 14" src="https://user-images.githubusercontent.com/8624777/113703408-15a3e980-96db-11eb-907a-692e033d4022.png">

**Comment by helen-ndip** [comment]
I do not have any categories yet but I  see that now it is gone. It is probably sporadic. I will look at it.

**Comment by helen-ndip** [comment]
I noticed that when I use a branch that was not updated with the main upstream develop branch, I did not see the default language but when updated with upstream develop the language shows. Could you please pull the recent changes from develop and try again, it could be a recent commit fixed it.

**Comment by Angamanga** [comment]
@helen-ndip Very odd, the bug still remains for me with the latest changes in `develop`. What browser are you using?

**Comment by Angamanga** [comment]
@helen-ndip How are you getting on?

**Comment by helen-ndip** [comment]
> @helen-ndip How are you getting on?

I finally could replicate the issue. When I run the application and navigate directly to creating the categories, I do not see the language selected but when I refresh it appears, so maybe I was refreshing the page and that is why I did not get the empty select.
![image](https://user-images.githubusercontent.com/72542706/114615770-f9391b80-9c9d-11eb-85d0-07125a290462.png)


I have made the fixes on categories and also creating a survey because I noticed it had the same issue. I also made the default language selected on the language selected in the sidebar.

![image](https://user-images.githubusercontent.com/72542706/114615337-787a1f80-9c9d-11eb-8d3c-a1efd9c39f73.png)
